libraries/cbor/README.md

```diff
@@ -0,0 +1,63 @@
+# CBOR Parsing Library
+
+This crate implements Concise Binary Object Representation (CBOR) from [RFC
+7049](https://datatracker.ietf.org/doc/html/rfc7049).
```

Comment:
I just realized that the old RFC is linked. [This is the new one](https://datatracker.ietf.org/doc/html/rfc8949).

Also [CTAP links the new RFC](https://fidoalliance.org/specs/fido-v2.1-rd-20210309/fido-client-to-authenticator-protocol-v2.1-rd-20210309.html#message-encoding).

Reply:
Fixed.